tools/astarte_import/lib/astarte/import/populatedb.ex

```diff
@@ -487,6 +487,62 @@ defmodule Astarte.Import.PopulateDB do
     end
   end
 
+  defp to_native_type(value_chars, :doublearray) do
+    {:ok,
+     Enum.map(value_chars, fn element ->
+       {:ok, value} = to_native_type(element, :double)
+       value
+     end)}
```

Comment:
Nit: this is a bit difficult to read as the tuple adds an unnecessary additional level of indentation. Consider using
```suggestion
    value =  Enum.map(value_chars, fn element ->
       {:ok, value} = to_native_type(element, :double)
       value
     end)
   {:ok, value}
```

It applies also to the other functions below